src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseConfirmsNotToProceedRespondentNotificationHandler.java

```diff
@@ -26,9 +27,11 @@
 
     private static final List<CaseEvent> EVENTS = List.of(
         NOTIFY_RESPONDENT_SOLICITOR1_FOR_CLAIMANT_CONFIRMS_NOT_TO_PROCEED,
+        NOTIFY_RESPONDENT_SOLICITOR2_FOR_CLAIMANT_CONFIRMS_NOT_TO_PROCEED,
         NOTIFY_RESPONDENT_SOLICITOR1_FOR_CLAIMANT_CONFIRMS_NOT_TO_PROCEED_CC);
 
     public static final String TASK_ID = "ClaimantConfirmsNotToProceedNotifyRespondentSolicitor1";
+    public static final String Task_ID_RespondentSol2 = "ClaimantConfirmsNotToProceedNotifyRespondentSolicitor2";
```

Comment:
Could we please keep it consistent with uppercase constants here. and also rename TASK_ID to TASK_ID_RESPONDENTSOL1 to clearly distinguish between the two

Reply:
Done